pom.xml

```diff
@@ -4,7 +4,24 @@
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-storage-parent</artifactId>
   <packaging>pom</packaging>
-  <version>2.11.4-SNAPSHOT</version><!-- {x-version-update:google-cloud-storage:current} -->
+  <version>2.11.4-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-core</artifactId>
+        <scope>test</scope>
+      </dependency>
+    </dependencies><!-- {x-version-update:google-cloud-storage:current} -->
```

Comment:
We don't want this here, was it supposed to go into `samples/pom.xml`?

Reply:
whoops .... might have fat fingered an intellij button. Reverting.